elastic-analytics/features/elastic-analytic.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const axios = require('axios')
+const { Then } = require('@cucumber/cucumber')
+
+const ANALYTICS_PROTOCOL = process.env.ANALYTICS_PROTOCOL || 'http'
+const ANALYTICS_HOSTNAME = process.env.ANALYTICS_HOSTNAME || 'localhost'
+const KIBANA_API_PORT =
+  process.env.KIBANA_API_PORT || '5601'
+const ES_ANALYTICS_API_PORT =
+  process.env.ES_API_PORT || '9201'
+const JS_REPORT_API_PORT =
+  process.env.JS_REPORT_API_PORT || '5488'
+
+const sendRequest = port => axios({
+  url: `${ANALYTICS_PROTOCOL}://${ANALYTICS_HOSTNAME}:${port}`
+})
+
+Then('the JS Report service should be up and running', async () => {
+  console.log('\n\nChecking the status of the JS Report service\n\n')
+
+  try {
+    await sendRequest(JS_REPORT_API_PORT)
+  
+    console.log('JS Report service is up\n')
+  } catch (error) {
+    if (error.response.status != 500) {
+      console.log('JS Report service is up\n') 
```

Comment:
@bradsawadye Do we expect errors? Why not fail the test if there's an issue?

Reply:
@bradsawadye

Reply:
@MattyJ007, I did it this way to cater for 401 errors. Some services require authentication.

Reply:
@bradsawadye Could you please add a comment in the code to explain this. Currently, it looks dodgy because you're hiding an error.

Also, can you add something to the console logs when it is a weird status code i.e. `console.log('JS Report service is up and responded with ${error.response.status}\n') `